contents/blog/launch-week-universe-of-new-features.md

```diff
@@ -0,0 +1,78 @@
+date: 2022-03-22
+title: PostHog Launch Week I: A Universe of New Features
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ..
+featuredImageType: full
+author: ["andy-vandervell"]
+categories: ["Product updates"]
+---
+
+PostHog made huge strides in 2021. To name just a few landmarks, we: 
+
+- Raised a [$15 million Series B ahead of schedule](/blog/why-we-raised-a-15m-series-b-ahead-of-schedule)
+- Became one of [YC's top-valued companies](/blog/yc-top-companies)
+- Rebranded our website [in just four weeks](/blog/postmortem-rebrand)
+- Migrated from [postgresql to ClickHouse](/blog/how-we-turned-clickhouse-into-our-eventmansion)
+- Achieved product market fit for PostHog Scale
+
+All of this (and more) lead to huge growth. Users grew by 4x, the number of open source contributors doubled, we acquired numerous reference customers, and went from zero revenue to significant revenue.
+
+As an open-source product, we're also delighted to see huge growth in our Slack community - now at over 2,000 members - and GitHub stars, which now stand at over 7,000.
+
+But this is just the beginning. We've also been busy making PostHog broader, more comprehensive, and a more powerful tool for making great products. 
+
+To celebrate, starting on Monday, March 21 we'll be officially launching a new feature or initative every day for a week.
+
+This is... PostHog Launch Week I: A Universe of New Features.
+
+## Chapter I: Experimentation
+
+- **Releasing:** Monday, March 21
+- **Directed by:** xxxx
+- **Starring:** Neil Kakkar
+- **Introducing:** A/B testing, multivariate tests and statistical significance
+
+Critics are calling it "the most statistically signicant new feature since [a joke Engineers will get, help please!]"
+
+## Chapter II: Project Sparkle
+
+- **Releasing:** Tuesday, March 22
+- **Directed by:** xxxx
+- **Starring:** Karl-Askel Puulman and Guido Iaquinti
+- **Introducing:** Sharded ClickHouse and easier self-hosted deploymments
+
+"Incredibly polished" ⭐️⭐️⭐️⭐️⭐️ - Hedgehog Gazette
+
+"Shines like a diamond" ⭐️⭐️⭐️⭐️⭐️ - Roger Hogbert
+
+"Puulman and Iaquinti are spectacular" ⭐️⭐️⭐️⭐️⭐️ - Christian Hedgehog Monitor
+
+## Chapter III: Data Management
+
+- **Releasing:** Wednesday, March 23
+- **Directed by:** xxxxx
+- **Starring:** Alex Kim
+- **Introducing:** New Data Management UI, event definitions, definition tagging and many more
+
+"An inspiring tale about overcoming the terrifying prospect of a data explosion. Do not miss!" - An influencer who wasn't even paid to say this  
+
+## Chapter IV: Collaboration
+
+- **Releasing:** Thursday, March 24
+- **Directed by:** xxxxx
+- **Starring:** Paolo Damico
+- **Introducing:** 
+
+[this one needs filling out] 
```

Comment:
I need a little more info on what we're covering here to complete this

Reply:
More info here: https://github.com/PostHog/posthog.com/issues/3050

Reply:
Hmm, I wonder if there might be a bit of overlap between 'Collaboration' and 'Data Management' here as [Alex's blog on the data management features](https://github.com/PostHog/posthog.com/pull/3099) covers the tagging / categorisation stuff fairly.

FYI, @paolodamico and @charlescook-ph

Reply:
Let me work on a draft for the blog post and I think that should help bring some context here. I wouldn't trust my creative writing skills to come up with as funny stuff as you have above.